src/rust/uk.mangainua/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "mangainia-aidoku"
```

Comment:
```suggestion
name = "mangainia"
```

Reply:
Should I also remove it in Cargo.lock file?

Reply:
it should update automatically when you build it

Reply:
> it should update automatically when you build it

Done!